src/main/java/de/zalando/aruha/nakadi/controller/TopicsController.java

```diff
@@ -115,7 +112,7 @@ public TopicPartition getPartition(@PathVariable("topicId") final String topicId
 	}
 
 	@Timed
-	@RequestMapping(value = "/{topicId}/partitions/{partitionId}/events", method = RequestMethod.GET)
+	@RequestMapping(value = "/{topicId}/partitions/{partitionId}/events/one", method = RequestMethod.GET)
```

Comment:
@jkliff 
Yes, this was an endpoint for reading a single event, we actually don't have it in our API. So I will then remove it.